_site/documentation.html

```diff
@@ -1,4 +1,9 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
+<meta charset="utf-8" />
+
+<!-- Redirect page after 3 seconds -->
+<meta http-equiv="refresh" content="3;url=https://dtinit.org/" />
+
```

Comment:
lets make one more change to this file just to be *absolutely* sure the change is due to your diret modification and not a redirect_to plugin execution

eg: maybe:
```diff
- Download White Paper
+ Download the White Paper
```